.travis.yml

```diff
@@ -306,6 +310,12 @@ before_install:
         # this assumes TEST_THEANO runs in the same build as TEST_MATPLOTLIB
         conda install theano;
     fi
+
+  - if [[ "${TEST_SYMENGINE}" == "true" ]]; then
+        conda config --add channels conda-forge;
+        conda config --add channels symengine;
```

Comment:
How is the `symengine` channel updated? Should we only use `conda-forge`, and perhaps keep the `symengine` channel commented out, for quick debugging of PRs, but the master should depend on the conda-forge version?